pallets/phala/src/stakepool.rs

```diff
@@ -541,6 +543,35 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// manually assign rewards to pools by root for fixing issue 763
```

Comment:
Upper case just like regular english sentences. Also, it's not exactly "by root", rather the `MiningSwitchOrigin`.